packages/@webex/plugin-meetings/src/interceptors/locusRetry.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {Interceptor} from '@webex/http-core';
+
+const rateLimitExpiryTime = new WeakMap();
+/**
+ * @class
+ */
+export default class LocusRetryStatusInterceptor extends Interceptor {
+  /**
+   * @returns {LocusRetryStatusInterceptor}
+   */
+  static create() {
+    // @ts-ignore
+    return new LocusRetryStatusInterceptor({webex: this});
+  }
+
+  /**
+   * Handle response errors
+   * @param {Object} options
+   * @param {WebexHttpError} reason
+   * @returns {Promise<WebexHttpError>}
+   */
+  onResponseError(options, reason) {
+    if (reason.statusCode >= 500 && reason.statusCode < 600 && options.uri.includes('locus')) {
+      const hasRetriedLocusRequest = rateLimitExpiryTime.get(this);
+      const retryAfterTime = options.headers['retry-after'] || 2000;
```

Comment:
Why doesn't this do exponential backoff with a parameterized exponential backoff start and end?

why 2000?

Reply:
The requirement is **retry once only**, based on the `retry-after` header, or **IMMEDIATELY** after the 503 is received. We MUST place a short delay (1-3 seconds) otherwise the service does not have the time to correct its routing.

Reply:
IMO this should just do a normal exponential backoff if there is no retryAfterTime, (that could be a follow up PR - ill leave it to shivani to decide)

Reply:
the JIRA mentions the request needs to be retried only once again - which makes sense to me because service unavailable should be getting resolved w load balancing on the backend - which should be immediate in-csae of failover imo, but I can create another followup JIRA for this